### DISCUSSION TITLE
Kafka KRaft support

### DISCUSSION THREAD

**Discussion by ITheCorgi** [edenhill/librdkafka#4126] [General]
I have a docker setup for Kafka with Kraft enable settings and use Golang wrapper lib for librdkafka 1.9.2 ([confluent-kafka-go](https://github.com/confluentinc/confluent-kafka-go))
According to this doc ([KIP-833](https://cwiki.apache.org/confluence/display/KAFKA/KIP-833%3A+Mark+KRaft+as+Production+Ready)) KRAFT is production ready feature. But when I try to produce a message into topic, I get the following error:
`|METADATA|rdkafka#producer-1| [thrd:main]: localhost:9093/bootstrap: Metadata request failed: refresh unavailable topics: Local: Required feature not supported by broker (0ms): Permanent`

Is Kraft supported by librdkafka or its in a plan to?

**Comment by mavam** [comment]
I have the same question and would be grateful for an answer.
